spec/features/sales_log_spec.rb

```diff
@@ -38,16 +38,16 @@
           visit("/sales-logs")
           click_button("Create a new sales log")
           click_link("Set up this sales log")
-          fill_in("sales_log[saledate(1i)]", with: "2022")
-          fill_in("sales_log[saledate(2i)]", with: "08")
-          fill_in("sales_log[saledate(3i)]", with: "10")
+          fill_in("sales_log[saledate(1i)]", with: Time.zone.today.year)
```

Comment:
it can be, I slightly prefer not freezing time where it's possible/easy as that makes it more likely we'll catch genuine bugs in the future, I don't see this behaviour as being limited to 2022.